examples/simple_example/main.tf

```diff
@@ -14,9 +14,27 @@
  * limitations under the License.
  */
 
-module "tags" {
-  source = "../.."
+module "tag" {
```

Comment:
Is it possible to create a storage bucket and attach location binding to the storage bucket? This will ensure that the  module is working for both scenarios.